config.example.yaml

```diff
@@ -0,0 +1,71 @@
+# Application Configuration Example
+
+# Database Configuration
+db:
+  # Database type (default: sqlite)
+  type: sqlite
+  # Path to the database schema
+  schema_path: "./database/migrations"
+  # Data Source Name
+  dsn: "file:./data/blockheaders.db?_foreign_keys=true&pooling=true"
+  # Database file path
+  file_path: "./data/blockheaders.db"
+  # Whether prepared DB is enabled
+  prepared_db: false
+  # Path to prepared database file
+  prepared_db_file_path: "./data/blockheaders.csv.gz"
+
+# P2P Configuration
+p2p:
+  # Maximum number of peers (inbound and outbound)
+  max_peers: 125
+  # Maximum number of peers per IP
+  max_peers_per_ip: 5
+  # Duration to ban misbehaving peers
+  ban_duration: 24h
+  # Minimum sync peer network speed
+  min_sync_peer_network_speed: 51200
```

Comment:
I'm not sure about exposing this config (and by "not sure" I mean, I would suggest to not exposing it)

Reply:
moved outside of the config, to a consts